frontend/src/scenes/insights/ActionFilter/ActionFilterRow.js

```diff
@@ -307,19 +323,36 @@ function MathSelector({ math, index, onMathSelect, areEventPropertiesNumericalAv
                             >
                                 {name}
                             </Tooltip>
-                        </Menu.Item>
+                        </Select.Option>
                     )
                 })}
-            </Menu>
-        )
-    }
-
-    return (
-        <Dropdown overlay={overlay}>
-            <Button className="ant-btn-md" data-attr={`math-selector-${index}`}>
-                {MATHS[math || 'total']?.name} <DownOutlined />
-            </Button>
-        </Dropdown>
+            </Select.OptGroup>
+            <Select.OptGroup key="property aggregates" label="Property aggregation">
+                {PROPERTY_MATH_ENTRIES.map(([key, { name, description, onProperty }]) => {
+                    const disabled = onProperty && !areEventPropertiesNumericalAvailable
+                    return (
+                        <Select.Option key={key} value={key} data-attr={`math-${key}-${index}`} disabled={disabled}>
+                            <Tooltip
```

Comment:
Right now the tooltips appear behind the menu rather than in front
![Uploading image.png…]()

Reply:
lmk if the latest changes fixes it. my local environment doesn't have this problem. The fix I put in isn't the most satisfying fix because the left edge of the tooltip might still be slightly under the menu but I don't have full control of the select.option element. Alternative if it's still broken could you see if dom editing and setting the zindex to some big number brings it forward?

Reply:
Yep looks good